app/code/Magento/CatalogInventory/Helper/ProductStockIsQtyDecimal.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Magento\CatalogInventory\Helper;
+
+use Magento\CatalogInventory\Api\StockItemRepositoryInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * Class GetProductIsQtyDecimalService
+ */
+class ProductStockIsQtyDecimal
+{
+    /**
+     * @var StockItemRepositoryInterface
+     */
+    protected $stockItemRepository;
+
+    /**
+     * @param StockItemRepositoryInterface $stockItemRepository
+     *
+     * @return void
```

Comment:
`@return` annotation for the `__construct` is not required. Additionally, constructors usually return instance of the class they are constructing.